packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -317,9 +324,62 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
   }
 
   a11yStatusTimeout: any
+  charKeysPressedTimeout: any
```

Comment:
it can be `number` type as `setTimeout` returns number

Reply:
It can't because it conflicts with type for `NodeJS.Timer`: https://github.com/Microsoft/TypeScript/issues/842
Obvious usage of `window.setTimeout()` solves the issue